### DISCUSSION TITLE
Open Source Roadmaps

### DISCUSSION THREAD

**Discussion by sid-am-ahd935** [scaleracademy/open-source-projects#132] [Ideas]
## Open Source Roadmaps for beginners to stay updated of their path of development

I have always been lost when it comes to contributing in open source. Not because the lack of the options/ideas, but because of the excess of it. All the beginners need to keep themselves in check of what they should do next in order to get ahead in the field of development. Every one has their own SDE sheets, but very few of them out there has a proper structured roadmap of technologies and languages to cover next in order to firm their grip into open source.

**Comment by BhumikaJena** [comment]
Nice idea!
